test/plot-schema.json

```diff
@@ -7597,6 +7633,18 @@
       },
       "role": "object"
      },
+     "ticklabelshiftx": {
```

Comment:
I thought you didn't want to implement this for `gl3d.scene.(x|y|z)axis`.
Is this working?

Reply:
Actually, you're right. I forgot about how to add ticklabel properties to cartesian plots exclusively. I have now removed the properties from all plots that are not cartesian.

If support for them is needed in another plot type I think it will be easy to open a new PR for that.